### PR TITLE
Support tags color in `TagsInput`

### PR DESCRIPTION
**Pull request by WilgnerFSDev**
## Description

This PR introduces two new feature for TagsInput component, the `->color()` method (HasColor Trait) to change the color of the tags/badges and the method `->inlineInput()` to make the input inline, after the badges/tags.

- [X] Visual changes (if any) are shown using screenshots/recordings of before and after.

**Before:**
tagsinput_before

**After:**
tagsinput_after

## Code style

- [X] `composer cs` command has been run.

## Testing

- [X] Changes have been tested.

## Documentation

- [X] Documentation is up-to-date.

### CODE REVIEW

**Comment by danharrin** [comment]
I'm leaning towards saying no to the inline method, but I'll let Zep have the final call

File: packages/forms/resources/views/components/tags-input.blade.php
Comment (zepfietje) [inline packages/forms/resources/views/components/tags-input.blade.php]: Move this up to be the first attribute.

File: packages/forms/resources/views/components/tags-input.blade.php
Comment (zepfietje) [inline packages/forms/resources/views/components/tags-input.blade.php]: Sort this alphabetically.

**Review by zepfietje — CHANGES_REQUESTED** [review]
Thanks for this PR, @WilgnerFSDev!

Could you revert the "inline" changes?
I think there's more edge cases to it that need to be handled, so let's hold off on that part.

@danharrin, regarding the color API. Wouldn't it be better to support per-tag colors as we do on text columns and entries?

**Comment by danharrin** [comment]
The tags are stored in Alpine on the frontend so we should not support per-tag colors ATM.

**Comment by WilgnerFSDev** [comment]
> Thanks for this PR, @WilgnerFSDev!
> 
> Could you revert the "inline" changes? I think there's more edge cases to it that need to be handled, so let's hold off on that part.
> 
> @danharrin, regarding the color API. Wouldn't it be better to support per-tag colors as we do on text columns and entries?

Sure, is done.
So for now, I will create a custom field to meet the client's requirement because the TagsInput (default) in the repeater takes up too much space.

Thank you for your time @danharrin and @zepfietje

**Pull request merged**
